### PR TITLE
Fix missing batch size support in create_network

### PR DESCRIPTION
**Pull request by huikang**
Without batch size support, all ovn-set command will be issued in
one single command, which may exceed the limit of OVN (e.g., > 10k
networks in a single command line).

Signed-off-by: Hui Kang kangh@us.ibm.com

### CODE REVIEW

**Comment by huikang** [comment]
Hi, @l8huang , could you take a look at this PR? Thanks.

File: rally_ovs/plugins/ovs/scenarios/ovn.py [hunk 1]
```diff
@@ -231,13 +231,15 @@ def _delete_acl(self, lswitches):
     def _create_networks(self, network_create_args):
         physnet = network_create_args.get("physical_network", None)
         lswitches = self._create_lswitches(network_create_args)
+        batch = network_create_args.get("batch", 1)
```
Comment (l8huang) [inline rally_ovs/plugins/ovs/scenarios/ovn.py]: suggest to use the number of lswitches as default value:
batch = network_create_args.get("batch", len(lswitches))

**Comment by l8huang** [comment]
one small comments :)

**Comment by mestery** [comment]
This LGTM after addressing the comment from @l8huang, thanks!

**Pull request merged**